src/App.vue

```diff
@@ -50,6 +53,14 @@ export default class App extends Vue {
         Axios.defaults.headers.post['Accept-Language'] = this.currentLanguage;
     }
 
+    private get maintenanceActive(): boolean {
+        return this.mainStore.state.maintenancePageStatus;
+    }
+
+    private activeMaintenanceStatus(): void {
+        this.mainStore.actions.activeMaintenancePage();
```

Comment:
@gonzalo-plaza Maybe "activeMaintenancePage" can have more sense like this: "activateMaintenancePage"